scipy/integrate/_quadrature.py

```diff
@@ -522,7 +523,7 @@ def _basic_simpson(y, start, stop, x, dx, axis):
 
 # Note: alias kept for backwards compatibility. simps was renamed to simpson
 # because the former is a slur in colloquial English (see gh-12924).
-def simps(y, x=None, dx=1.0, axis=-1, even=None):
+def simps(y, x=None, dx=1.0, axis=-1, even=_NoValue):
     """An alias of `simpson`.
 
     `simps` is kept for backwards compatibility. For new code, prefer
```

Comment:
Sidenote: this alias sounds like it should eventually be removed. Some digging revealed it came from https://github.com/scipy/scipy/issues/12924, making it even more of a candidate for deprecation and removal.

Reply:
https://github.com/scipy/scipy/issues/18702